server/app.js

```diff
@@ -46,6 +46,34 @@ app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
 
+const upload = require("./middleware/multer");
+const cloudinary = require("./config/cloudinaryConfig");
+const fs = require("fs");
+app.use("/upload", upload.array("picture"), async (req, res) => {
```

Comment:
you might want to move this into it's own upload route so you don't crowd `app.js`